### DISCUSSION TITLE
Add another option like --continue but respect the dependency graph

### DISCUSSION THREAD

**Discussion by tomdee** [vercel/turborepo#8213] [Ideas]
### Goals

Continue running tasks, but only if all dependent tasks are successfull

### Non-goals

1.
2.
3.

### Background

Right now `--continue` is all or nothing. It runs all tasks, even ones which don't make sense.

### Proposal

Maybe allow an extra flag for continue, e.g. `--continue=only-safe`

**Comment by anthonyshew** [comment]
Duplicate of #7461. Sounds like you're not alone! 😄
